package.json

```diff
@@ -57,7 +57,7 @@
         "babel-plugin-const-enum": "^1.1.0",
         "copy-webpack-plugin": "~9.0.1",
         "css-loader": "^3.2.0",
-        "cssnano": "^4.1.10",
+        "cssnano": "^5.0.11",
```

Comment:
Are you sure we're still using this dependency? When I do a global search for `cssnano` I get nothing, looks like we would see it in one of the config files? https://cssnano.co/docs/getting-started/

Reply:
I did the same, didn't find it, but didn't feel like removing it and accidentally breaking something, since there are zero code changes happening here.  I don't know what cssnano is supposed to do, or how it folds in.  I'll check that link.

Reply:
Took a look, yeah maybe we can remove it.